itunesiap/request.py

```diff
@@ -89,7 +90,8 @@ def verify(self, **options):
             try:
                 response = self.verify_from(RECEIPT_PRODUCTION_VALIDATION_URL, verify_ssl)
             except exceptions.InvalidReceipt:
-                if not use_sandbox:
+                if not use_sandbox and (response is None or
```

Comment:
`response` will be always `None` in this case. So this patch doesn't fix the behavior of the code. (Because it is always `True`)

Instead of it, `InvalidReceipt` will have a property `status`. So how about to try: 
```
if not use_sandbox and e.status != STATUS_SANDBOX_RECEIPT_ERROR:
```